rust-src/concordium_base/src/id/sigma_protocols/com_eq_sig.rs

```diff
@@ -17,14 +20,14 @@ use itertools::izip;
 use rand::*;
 
 #[derive(Clone, Debug, Serialize)]
-pub struct Witness<P: Pairing, C: Curve<Scalar = P::ScalarField>> {
-    /// The witness that the prover knows $r'$ (see specification)
-    witness_rho:    P::ScalarField,
-    /// List of witnesses $(w_i, R_i)$ that the user knows the messages m_i and
+pub struct Response<P: Pairing, C: Curve<Scalar = P::ScalarField>> {
+    /// The response that the prover knows $r'$ (see specification)
+    response_rho:    P::ScalarField,
+    /// List of responses $(w_i, R_i)$ that the user knows the messages m_i and
```

Comment:
If I understand correctly, we call the responses as `(res_m, res_r)` later. So, this should also be named consistently. 
```suggestion
    /// List of responses $(res_m_i, res_r_i)$ that the user knows the messages m_i and
```